### PR TITLE
Unit tests counter

### PR DESCRIPTION
**Pull request by mhs5218**
Completed Counter unit test, unable to get change and validate working

Edited onFocus function in counter.module.js so that tInputElem is assigned a value if it is undefined. Without this select on focus unit tests fail because tInputElem is undefined.

### CODE REVIEW

**Comment by jrh418** [comment]
Please:
- Break tests into smaller groups (i.e. "required attributes" for first 4 tests)
- Add tests that test the expected functionality (not limited to the following)
  - Min of 1 means that if the value is 1 and the "-" button is clicked the value should not change
  - Min larger than the max logs an error / fails to display
  - Mouse arrows can change values
  - Only integers are allowed to be entered
  - Basically, check the counter.module.js file for functionality and test all functions that make sense
